src/components/Dropdown/asyncDropdown/AsyncDropdown.jsx

```diff
@@ -130,6 +135,17 @@ class AsyncDropDown extends React.Component {
       onKeyDown: this.props.onKeyDown,
     };
 
+    const handleCreate = () => {
+      const newFunc = () => {
+        onCreate(this.state.inputValue);
+      };
+
+      if (children) {
+        this.setState({ inputValue: '' });
+      }
+      newFunc();
```

Comment:
Siden denne ikke egentlig returnerer noe, kan vi ikke forenkle den til noe sånt: (?)
```suggestion
      onCreate(this.state.inputValue);
      if (children) {
        this.setState({ inputValue: '' });
      }
```

Reply:
true true